pkg/requests/base.go

```diff
@@ -20,24 +20,24 @@ import (
 
 // RequestParameters captures the structure of the parameters that can be sent to Stripe
 type RequestParameters struct {
-	data          []string
-	expand        []string
-	startingAfter string
-	endingBefore  string
-	idempotency   string
-	limit         string
-	version       string
-	stripeAccount string
+	Data          []string
+	Expand        []string
+	StartingAfter string
+	EndingBefore  string
+	Idempotency   string
+	Limit         string
+	Version       string
+	StripeAccount string
 }
 
 // AppendData appends data to the request parameters.
 func (r *RequestParameters) AppendData(data []string) {
-	r.data = append(r.data, data...)
+	r.Data = append(r.Data, data...)
 }
 
 // SetStripeAccount sets the value for the `Stripe-Account` header.
 func (r *RequestParameters) SetStripeAccount(value string) {
-	r.stripeAccount = value
+	r.StripeAccount = value
```

Comment:
Why do we export these? Part of the reason we have the `AppendData` and `SetStripeAccount` functions to set the data without exporting the actual field.